index.html

```diff
@@ -0,0 +1,195 @@
+                        </li>
+                    </ul>
+                </div>
+                <div>
+                    <a class="btn-lg text-light" href="#" role="button">Join devfest 2023</a>
+                </div>
+            </div>
+        </section>
+        <section id="speakers" class="featured-speakers text-center pt-5 pb-5">
+            <div class="container">
+                <div class="main-title position-relative">
+                    <h2>Featured Speakers</h2>
+                </div>
+                <ul id="speakers-card" class="row pt-5 pb-5 ps-md-3 pe-md-3 ps-lg-3 pe-lg-3"></ul>
+            </div>
+        </section>
+        <section id="partners" class="Partners text-center pt-5 pb-5 gray-dk">
+            <div class="container">
+                <div class="main-title position-relative">
+                    <h2 class="text-light">Partners</h2>
+                </div>
+                <div>
+                    <ul class="row d-flex justify-content-center pt-5">
+                        <li class="col-md-auto p-2 m-2 col-4">
+                            <img src="img/wtm-logo.png" alt="google-ic" />
+                        </li>
+                        <li class="col-md-auto p-2 m-2 col-4">
+                            <img src="img/CIC-logo.png" alt="google-ic" />
+                        </li>
+                        <li class="col-md-auto pt-4 p-2 m-2 col-4">
+                            <img src="img/mozilla-logo.png" alt="google-ic" />
+                        </li>
+                        <li class="col-md-auto p-2 m-2 col-6">
+                            <img src="img/github-logo.png" alt="google-ic" />
+                        </li>
+                        <li class="col-md-auto pt-4 p-2 m-2 col-6">
+                            <img src="img/slack-logo.png" alt="google-ic" />
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+        <footer id="footer" class="row d-md-flex pt-4 pb-4 ps-lg-5 ps-md-2">
+            <div class="ms-md-3 col-md-auto col-6 text-center">
+                <img id="logo" class="ps-2 pt-4 gdg" src="img/logo_dark_gdg.svg" alt=""/>
+            </div>
+            <div class="col-md-auto col-6 pt-lg-2 pt-md-2 ps-md-3 ps-lg-4 pe-3">
+                <h6 class="fontS-12 pt-3 pt-md-0">
+                    Google Developers under terms that explicitly encourage people to
+                    take, modify, reuse, re-purpose, and remix our work as they see fit.
+                </h6>
+                <h6 class="fontS-12 d-none d-md-block">
+                    Proper attribution is required when you reuse or create modified
+                    versions of content that appears on a page made available under the
+                    terms of the Creative Commons Attribution license.
+                </h6>
+            </div>
+        </footer>
```

Comment:
- [ ] As mentioned in the about footer, kindly ensure that you center the home page footer as well. 
- [ ] On mobile, this section is [OPTIONAL] and should not be there according to the design guidelines. However, this can be ignored and having the footer in both mobile and desktop looks good :slightly_smiling_face:

Reply:
@Nemwel-Boniface  I used the same property that was previously used.

Reply:
Well resolved :heavy_check_mark: